twitter/src/com/treode/twitter/server/handler/package.scala

```diff
@@ -30,6 +35,20 @@ package object handler {
       rsp
     }
 
+    def apply (req: Request, notifications: Notification [Unit]): Response = {
+      val rsp = req.response
+      notifications match {
+        case errors @ Errors (_) =>
+          val mapper = new ObjectMapper() with ScalaObjectMapper
+          mapper.registerModule (DefaultScalaModule)
+          rsp.write(mapper.writeValueAsString(errors.strings))
```

Comment:
Make a Jackson serializer for Notification. For examples, look at the serializers in com.treode.jackson. You will need to add your serializer to the TreodeModule. Then you can just call `mapper.writeValueAsString (notification)`.